tools/cli/config.json

```diff
@@ -6,18 +6,18 @@
             "grpc_endpoint": "http://localhost:9090",
             "gas_amount": "0.025",
             "gas_denom": "uslay",
-            "address_kind": {"cosmos":{"prefix":"slay3r"}}
+            "address_kind": {"cosmos":{"prefix":"layer"}}
```

Comment:
It will be gone after rebase, and rebase will be gone after merge conflits are solved...